### PR TITLE
[Localization] Trick Room challenge translations

### PR DESCRIPTION
**Pull request by Adri1**
## What are the changes the user will see?
None, translation of pending PR #4343 

## Why am I making these changes?
To have it translated

## What are the changes from a developer perspective?
None

## Checklist
- [X] **I'm using `beta` as my base branch**
- [X] There is no overlap with another PR?
- [X] The PR is self-contained and cannot be split into smaller PRs?
- [X] Have I provided a clear explanation of the changes?

### CODE REVIEW

File: src/locales/pt_BR/achv.json [hunk 1]
```diff
@@ -269,6 +269,10 @@
     "name": "A torre da derrotA",
     "description": "Complete o desafio da Batalha Inversa.\n.asrevnI ahlataB ad oifased o etelpmoC"
   },
+  "TRICK_ROOM": {
+    "name": "You're Too Fast!",
+    "description": "Complete the Trick Room challenge."
```
Comment (josericardo-fo) [inline src/locales/pt_BR/achv.json:274]:
```suggestion
    "name": "Você é Muito Rápido!",
    "description": "Complete o desafio de Trick Room."
```

File: src/locales/pt_BR/challenges.json [hunk 2]
```diff
@@ -33,5 +33,12 @@
     "desc": "Fraquezas e resistências de tipos são invertidas e nenhum tipo é imune a outro tipo.\nDesativa as conquistas de outros desafios.",
     "value.0": "Desligado",
     "value.1": "Ligado"
+  },
+  "trickRoom": {
+    "name": "Trick Room",
+    "shortName": "Trick Room",
+    "desc": "All battles are under the effect of Trick Room, where slower Pokémon move sooner and faster Pokémon move later.\nDisables other challenges' achievements.",
+    "value.0": "Off",
+    "value.1": "On"
```
Comment (josericardo-fo) [inline src/locales/pt_BR/challenges.json:42]:
```suggestion
  "trickRoom": {
    "name": "Sala de Truques",
    "shortName": "Sala de Truques",
    "desc": "Todas as batalhas estão sob o efeito de Trick Room, onde Pokémon mais lentos atacam primeiro e Pokémon mais rápidos atacam depois.\nDesativa as conquistas de outros desafios.",
    "value.0": "Desligado",
    "value.1": "Ligado"
```

File: src/locales/pt_BR/achv.json [hunk 3]
```diff
@@ -269,6 +269,10 @@
     "name": "A torre da derrotA",
     "description": "Complete o desafio da Batalha Inversa.\n.asrevnI ahlataB ad oifased o etelpmoC"
   },
+  "TRICK_ROOM": {
+    "name": "Você é Muito Rápido!",
+    "description": "Complete o desafio de Trick Room."
```
Comment (josericardo-fo) [inline src/locales/pt_BR/achv.json:274]:
```suggestion
    "description": "Complete o desafio da Sala de Truques."
```

File: src/locales/ja/achv.json [hunk 4]
```diff
@@ -266,6 +266,10 @@
     "name": "カガミよミガカ",
     "description": "反転バトルチャレンジを　クリアする\nるすアリク　をジンレャチルトバ転反"
   },
+  "TRICK_ROOM": {
+    "name": "You're Too Fast!",
+    "description": "Complete the Trick Room challenge."
+  },
   "BREEDERS_IN_SPACE": {
     "name": "宇宙のブリーダー！",
     "description": "宇宙バイオームで　エキスパートブリーダーを　倒す"
```
Comment (Chapybara-jp) [inline src/locales/ja/achv.json:275]:
```suggestion
  "TRICK_ROOM": {
    "name": "速すぎだぜ！",
    "description": "トリックルームチャレンジを　クリアする"
  },
  "BREEDERS_IN_SPACE": {
    "name": "むしろブリーダー宇宙人？",
    "description": "宇宙バイオームで　ブリーダー名人を　倒す"
```

File: src/locales/ja/challenges.json [hunk 5]
```diff
@@ -32,5 +32,12 @@
     "desc": "タイプ相性が 反転で、なんの タイプも 「効果はなし」が ありません\n他の チャレンジの 実績が 無効に されます",
     "value.0": "オフ",
     "value.1": "オン"
+  },
+  "trickRoom": {
+    "name": "Trick Room",
+    "shortName": "Trick Room",
+    "desc": "All battles are under the effect of Trick Room, where slower Pokémon move sooner and faster Pokémon move later.\nDisables other challenges' achievements.",
+    "value.0": "Off",
+    "value.1": "On"
```
Comment (Chapybara-jp) [inline src/locales/ja/challenges.json:41]:
```suggestion
  },
  "trickRoom": {
    "name": "トリックルーム",
    "shortName": "トリルー",
    "desc": "バトルは 全て トリックルームの 影響が あるから、 すばやさの 低い順に 行動できる ように なる\n他の チャレンジの 実績が 無効に されます",
    "value.0": "オフ",
    "value.1": "オン"
```

File: src/locales/zh_CN/achv.json [hunk 6]
```diff
@@ -273,6 +273,10 @@
     "name": "镜子子镜",
     "description": "完成逆转之战挑战\n战挑战之转逆成完"
   },
+  "TRICK_ROOM": {
+    "name": "You're Too Fast!",
+    "description": "Complete the Trick Room challenge."
```
Comment (40chyan) [inline src/locales/zh_CN/achv.json:278]:
```suggestion
    "name": "你太快了！",
    "description": "完成戏法空间挑战"
```

File: src/locales/zh_CN/challenges.json [hunk 7]
```diff
@@ -32,5 +32,12 @@
     "desc": "属性相克关系被反转，且没有任何属性对其他属性免疫。\n禁用其他挑战的成就。",
     "value.0": "关闭",
     "value.1": "开启"
+  },
+  "trickRoom": {
+    "name": "Trick Room",
+    "shortName": "Trick Room",
+    "desc": "All battles are under the effect of Trick Room, where slower Pokémon move sooner and faster Pokémon move later.\nDisables other challenges' achievements.",
+    "value.0": "Off",
+    "value.1": "On"
```
Comment (40chyan) [inline src/locales/zh_CN/challenges.json:41]:
```suggestion
    "name": "戏法空间",
    "shortName": "戏法空间",
    "desc": "所有战斗受到戏法空间影响\n速度更慢的宝可梦会先行动\n使其他的挑战无效。",
    "value.0": "开启",
    "value.1": "关闭"
```

File: src/locales/de/achv.json [hunk 8]
```diff
@@ -284,6 +284,10 @@
     "name": "Spieglein, Spieglein an der Wand",
     "description": "Schließe die 'Umkehrkampf' Herausforderung ab"
   },
+  "TRICK_ROOM": {
+    "name": "You're Too Fast!",
+    "description": "Complete the Trick Room challenge."
+  },
```
Comment (CodeTappert) [inline src/locales/de/achv.json:290]:
```suggestion
  "TRICK_ROOM": {
    "name": "Halt Stop!",
    "description": "Schließe die 'Bizarroraum' Herausforderung ab."
  },
```

File: src/locales/es/achv.json [hunk 9]
```diff
@@ -175,6 +175,10 @@
     "name": "Espejo ojepsE",
     "description": "Completa el reto de Combate Inverso.\n.osrevnI etabmoC ed oter le atelpmoC"
   },
+  "TRICK_ROOM": {
+    "name": "You're Too Fast!",
+    "description": "Complete the Trick Room challenge."
+  },
```
Comment (DanStevensonx) [inline src/locales/es/achv.json:181]:
```suggestion
    "name": "Espejo ojepsE",
    "description": "Completa el desafío de Combate Inverso.\n.osrevnI etabmoC ed oter le atelpmoC"
  },
  "TRICK_ROOM": {
    "name": "¡Eres demasiado rápido!",
    "description": "Completa el desafío Espacio Raro."
  },
```

**Review by DanStevensonx — COMMENTED** [review]
Translated to Spanish

File: src/locales/ko/achv.json [hunk 10]
```diff
@@ -265,6 +265,10 @@
     "name": "상성 전문가(였던 것)",
     "description": "거꾸로 배틀 챌린지 모드 클리어."
   },
+  "TRICK_ROOM": {
+    "name": "You're Too Fast!",
+    "description": "Complete the Trick Room challenge."
```
Comment (KimJeongSun) [inline src/locales/ko/achv.json:270]:
```suggestion
    "name": "하지만 빨랐죠?",
    "description": "트릭룸 챌린지 모드 클리어."
```

File: src/locales/ko/challenges.json [hunk 11]
```diff
@@ -33,5 +33,12 @@
     "desc": "타입 상성이 반대로 바뀌고 면역 타입은 약점 타입이 됩니다.\n설정 시 다른 챌린지 업적은 달성할 수 없습니다.",
     "value.0": "해제",
     "value.1": "설정"
+  },
+  "trickRoom": {
+    "name": "Trick Room",
+    "shortName": "Trick Room",
+    "desc": "All battles are under the effect of Trick Room, where slower Pokémon move sooner and faster Pokémon move later.\nDisables other challenges' achievements.",
+    "value.0": "Off",
+    "value.1": "On"
```
Comment (KimJeongSun) [inline src/locales/ko/challenges.json:42]:
```suggestion
    "name": "트릭룸",
    "shortName": "트릭룸",
    "desc": "모든 배틀은 트릭룸 상태로 진행됩니다. 더 빠른 포켓몬은 늦게, 더 느린 포켓몬이 먼저 행동합니다.\n설정 시 다른 챌린지 업적은 달성할 수 없습니다.",
    "value.0": "해제",
    "value.1": "설정"
```

**Review by Chapybara-jp — DISMISSED** [review]

File: src/locales/it/achv.json [hunk 12]
```diff
@@ -261,9 +261,9 @@
     "name": "Buona la prima!",
     "description": "Completa la modalità sfida 'Un nuovo inizio'."
   },
-  "BREEDERS_IN_SPACE": {
-    "name": "Breeders in Space!",
-    "description": "Beat the Expert Pokémon Breeder in the Space Biome."
+  "TRICK_ROOM": {
+    "name": "You're Too Fast!",
+    "description": "Complete the Trick Room challenge."
```
Comment (NicusPulcis) [inline src/locales/it/achv.json:266]:
```suggestion
  "TRICK_ROOM": {
    "name": "Vai troppo veloce!",
    "description": "Completa la sfida 'Distortozona'."
```

File: src/locales/it/challenges.json [hunk 13]
```diff
@@ -25,5 +25,12 @@
     "desc": "Puoi usare solo gli starter originali, e come se avessi appena cominciato Pokérogue.",
     "value.0": "Off",
     "value.1": "On"
+  },
+  "trickRoom": {
+    "name": "Trick Room",
+    "shortName": "Trick Room",
+    "desc": "All battles are under the effect of Trick Room, where slower Pokémon move sooner and faster Pokémon move later.\nDisables other challenges' achievements.",
+    "value.0": "Off",
+    "value.1": "On"
```
Comment (NicusPulcis) [inline src/locales/it/challenges.json:34]:
```suggestion
  },
  "trickRoom": {
    "name": "Distortozona",
    "shortName": "Distortozona",
    "desc": "Tutte le battaglie subiscono l'effetto della Distortozona: i Pokémon più lenti sono più veloci e viceversa.\nCiò disattiva gli obiettivi per le altre sfide.",
    "value.0": "Off",
    "value.1": "On"
```

**Review by NicusPulcis — APPROVED** [review]
italian localized

**Comment by innerthunder** [comment]
@Adri1 is this still needed with the locale migration?

**Comment by Adri1** [comment]
> @Adri1 is this still needed with the locale migration?

No, but as long as the PR tied to this one don't show any new sign of activity nor the dev of it decided to added the English keys to the locale repo, I keep it alive as it contains pending translations.
I'll delete it only after I can copy-paste it in Pontoon

**Comment by Adri1** [comment]
@KimJeongSun You can delete the locales from your PR, I added all of them here:
https://github.com/pagefaultgames/pokerogue-locales/pull/32

**Pull request closed without merging**